cmd/server.go

```diff
@@ -151,8 +203,26 @@ func initServerCmd() {
 		Args:    cobra.ExactArgs(1),
 		Run:     produceServerCmdRunFunc(client, serverDeleteAction),
 	}
+	var createCmd = &cobra.Command{
+		Use:     "create [flags]",
+		Example: `./gscloud server create --name "Server Name" --cpu 6 --mem 4 --with-template "Debian 10" --password "p4ssw0rd" --hostname "gridscale"`,
+		Aliases: []string{"create"},
+		Short:   "Create server",
+		Long:    `Create a new server.`,
+		Run:     produceServerCmdRunFunc(client, serverCreateAction),
+	}
 
+	// Flags
 	serverOffCmd.PersistentFlags().BoolVarP(&forceFlag, "force", "f", false, "Force shutdown (no ACPI)")
-	serverCmd.AddCommand(serverLsCmd, serverOnCmd, serverOffCmd, removeCmd)
+	createCmd.PersistentFlags().StringVarP(&servernameFlag, "name", "n", "default_server", "Define servername")
+	createCmd.PersistentFlags().IntVarP(&cpuFlag, "cpu", "c", 1, "Define cpu cores")
+	createCmd.PersistentFlags().IntVarP(&memFlag, "mem", "m", 1, "Define Memory")
+	createCmd.PersistentFlags().StringVarP(&passwordFlag, "password", "p", "pass", "Define password")
+	createCmd.PersistentFlags().StringVarP(&hostnameFlag, "hostname", "H", "no_hostname", "Define hostname")
+	createCmd.PersistentFlags().StringVarP(&templateFlag, "with-template", "t", "Debian 10", "Define Operating System")
+	createCmd.PersistentFlags().IntVarP(&storageFlag, "storage-size", "s", 10, "Storage size")
```

Comment:
Better do not give defaults for flags here. We would overwrite API defaults and it is really hard to come up with sensible defaults so lets better not do this.

Reply:
You're right, hadn't thought that far. I agree with you.